.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: rust
 
+# Disabled cargo cache because it causes conflicts with the CentOS container build.
 # hopefully prevent the cache from becoming huge - https://levans.fr/rust_travis_cache.html
-before_cache:
-  - rm -rf /home/travis/.cargo/registry
+# before_cache:
+#   - rm -rf /home/travis/.cargo/registry
```

Comment:
If we aren't going to cache, I think we should just remove the related commented out cargo lines

Reply:
I'd like to figure out why this broke stuff, but realistically you're right; I'll just disable it.